tests/test__minhash.py

```diff
@@ -949,3 +951,27 @@ def test_minhash_abund_merge_flat_2():
         b.add_hash(i)
 
     a.merge(b)
+
+
+def test_distance_matrix(track_abundance):
```

Comment:
This test doesn't check the actual code in `commands.py`, but reimplements the behavior in new code. As such I don't think it's particularly useful in the long term, because at some point we may well change the code in `commands.py` at which point this test would no longer be relevant (although it's reassuring that when you copy-paste the current code, it does work).

Reply:
Fair point, I was trying to make sure the symmetry of the distance matrix is not lost (we had problems before when `E.similarity(E1)` was not the same as `E1.similarity(E)`